tests/specs/tax-calculation/test-order-tax-calculation-validator.php

```diff
@@ -20,7 +19,7 @@ public function setUp() {
 		$this->mock_order = $this->createMock( WC_Order::class );
 		$this->mock_order->method( 'get_subtotal' )->willReturn( 1.0 );
 		$this->mock_tax_request_body = $this->createMock( Tax_Request_Body::class );
-		$this->mock_nexus            = $this->createMock( WC_Taxjar_Nexus::class );
+		$this->mock_nexus            = $this->createMock( \WC_Taxjar_Nexus::class );
```

Comment:
Just wondered why you removed the "use" import here, if it was on purpose.

Reply:
Nope it was not intentional. I will revert it.